src/tools/ScenarioMeasurement/Startup/GenericStartupParser.cs

```diff
@@ -40,7 +40,7 @@ public IEnumerable<Counter> Parse(string mergeTraceFile, string processName, ILi
 
                 source.Kernel.ProcessStart += evt =>
                 {
-                    if (processName.Equals(evt.ProcessName, StringComparison.OrdinalIgnoreCase) && pids.Contains(evt.ProcessID) && evt.CommandLine == commandLine)
+                    if (processName.Equals(evt.ProcessName, StringComparison.OrdinalIgnoreCase) && pids.Contains(evt.ProcessID) && evt.CommandLine.Trim() == commandLine.Trim())
```

Comment:
Would it break here without trim?  If so, can we do trim in Startup.cs so we don't need to add this code to every parser?

Reply:
The erroneous space is on the event payload, so it should be here. I'm trimming both out of paranoia. I'll clean this up when I refactor this code.